sdks/java/core/src/main/java/org/apache/beam/sdk/options/StreamingOptions.java

```diff
@@ -29,5 +29,10 @@
   @Hidden
   boolean isStreaming();
 
+  @Description("If true will use Streaming Engine.")
+  boolean isEnableStreamingEngine();
```

Comment:
Should this be in GcpOptions (https://github.com/apache/beam/blob/762abacb2c91fb86e6ac7d2e1fbc874e832ee2a9/sdks/java/extensions/google-cloud-platform-core/src/main/java/org/apache/beam/sdk/extensions/gcp/options/GcpOptions.java) ?

Is this file for generic streaming options that can apply to any runner?

Reply:
I believe this should be in GcpOptions

Reply:
I agree I've moved it.